code/script.js

```diff
@@ -1,47 +1,150 @@
+//Startup greeting after load (timeout 1000)
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Good day Sir/Ms, please type your name`, 'bot')
 }
 
-// Set up your eventlisteners here
+// First function handle name input
+const handleNameInput =  (event) => {
+  event.preventDefault()
+  const name = inputText.value
+  showMessage(`My name is ${name}`, 'user')
+  inputText.value = ''
+  setTimeout(() => destination(), 1000)
+}
+
+//Second function choose of destination
+const destination = () => {
+  showMessage(`Choose your destination here`, 'bot')
+  inputWrapper.innerHTML = `
+    <button id="gotBtn">Gothenburg</button>
+    <button id="mmxBtn">Malmö</button>
+    <button id="stoBtn">Stockholm</button>`
 
+  document
+    .getElementById('gotBtn')
+    .addEventListener('click', () => { showMessage('Gothenburg', 'user')
+    setTimeout(() => showMessage('Gothenburg is a nice place, choose your vehicle', 'bot'), 1000)
+    inputWrapper.innerHTML = ''
+      setTimeout(vehicle, 2000)
+  })
+
+  document
+    .getElementById('mmxBtn')
+    .addEventListener('click', () => { showMessage('Malmö', 'user')
+    setTimeout(() => showMessage('Malmö is a nice place, choose your vehicle', 'bot'), 1000)
+    inputWrapper.innerHTML = ''
+    setTimeout(vehicle, 2000)
+  })
+    
+  document
+    .getElementById('stoBtn')
+    .addEventListener('click', () => { showMessage('Stockholm', 'user')
+    setTimeout(() => showMessage('Stockholm is a nice place, choose your vehicle', 'bot'), 1000)
+    inputWrapper.innerHTML = ''
+    setTimeout(vehicle, 2000)
+  })
+}
+
+//Third function choose of vehicle
+const vehicle = () => {
+  inputWrapper.innerHTML = `
+    <select id="select">
+      <option value="" selected disabled>👇 Select vehicle</option>
+      <option value="Limo">Limousine</option>
+      <option value="Minibus">Minibus</option>
+      <option value="Rickshaw">Rickshaw</option>
+    </select>`
+
+  document.getElementById("select").addEventListener('change', () => { 
+    showMessage(`Please take me in a ${select.value}`, 'user')
+    setTimeout(rate,1000)
+  })
+}
+
+//Fourth function decide rate
+const rate = () => {
+  let price
+  if (select.value === 'Limo') {
+    price = '20000SEK'
+  } else if (select.value === 'Minibus') {
+    price = '5000SEK'
+  } else {
+    price = '10sek'
+    alert('Unfortunately your ride will be in a Rickshaw')
+  }
+  showMessage(`${select.value}, that will be ${price}. Would you like to place the order?`, 'bot')
+  inputWrapper.innerHTML = ''
+
+  inputWrapper.innerHTML = `
+    <button id="Yes">Yes</button>
+    <button id="No">No</button>`
+
+    document.getElementById("Yes").addEventListener("click", () => {
+      showMessage('Yes', 'user')
+      setTimeout(order, 1000)
+    })
+    document.getElementById("No").addEventListener("click", () => {
+      location.reload()
+      return false  
+    })
```

Comment:
Nice touch with a possibility to reload and choose another destination and vehicle!